src/conversion_helper.js

```diff
@@ -903,22 +965,41 @@ function formatConversion(conversions) {
     const roundedConversions = conversion['rounded'];
     if (Array.isArray(roundedConversions)) {
       conversion['formatted'] = roundedConversions.map(rc => {
-        return createMap(rh.addCommas(rc['number']), rc['unit']);
+        let result = {
+          'numbers': [],
```

Comment:
This can be inlined to something like `'numbers': rc['numbers'].map(i => rh.addCommas(i))`